src/test/java/service/OperationTest.java

```diff
@@ -0,0 +1,29 @@
+package service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import model.FruitTransaction;
+import org.junit.jupiter.api.Test;
+import strategy.Operation;
+
+class OperationTest {
+    private static final FruitTransaction validFruit =
+            new FruitTransaction("b", "banana", 100);
+    private static final FruitTransaction invalidOperationField =
+            new FruitTransaction("q", "banana", 100);
```

Comment:
we don't test models, they are considered tested implicitly when we test services that use them